### PR TITLE
v1.0.0-rc.4 release

### PR DESCRIPTION
**Pull request by fhunleth**

### CODE REVIEW

**Comment by fhunleth** [comment]
I'd like to get another rc out since we're asymptotically approaching 1.0.0.

**Comment by ConnorRigby** [comment]
So... i thought this was already out and i built my system with it already :100:

**Comment by fhunleth** [comment]
I don't think that "bleeding" describes the edge that you live on. In any case, this is a pretty minor bump for everyone. I'm going to leave it open a little longer so that @mobileoverlord has a chance to get in any Elixir tooling fixes that I don't know about. I really hope this is the final rc before 1.0...

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
 
+## v1.0.0-rc.4
+
+* New features
+  * Buildroot 2018.02.1 patch release integrated. This removes the need for two
+    patches that we had been maintaining and includes a few minor version bugs
+    bug fixes.
+  * Refactor the rootfs_overlay merge logic to support multiple rootfs_overlays.
+    Changes are required in `nerves` to make this usable, but the restriction is
+    longer due to nerves_system_br.
```
Comment (tmecklem) [inline CHANGELOG.md]: Should this say "the restriction is no longer due to nerves_system_br"?

Comment (ConnorRigby) [reply CHANGELOG.md]: also fwiw that change is [here](https://github.com/nerves-project/nerves/pull/269)

Comment (fhunleth) [reply CHANGELOG.md]: Thanks! Fix the text and added the link.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
 
+## v1.0.0-rc.4
+
+* New features
+  * Buildroot 2018.02.1 patch release integrated. This removes the need for two
+    patches that we had been maintaining and includes a few minor version bugs
+    bug fixes.
+  * Refactor the rootfs_overlay merge logic to support multiple rootfs_overlays.
+    To use this in your programs, a corresponding update is required in the
+    `nerves` project. See https://github.com/nerves-project/nerves/pull/269.
```
Comment (ConnorRigby) [inline CHANGELOG.md]: `[the pull request](https://github.com/nerves-project/nerves/pull/269)`?

**Pull request merged**